cmd/podman/network_create.go

```diff
@@ -0,0 +1,69 @@
+// +build !remoteclient
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	networkcreateCommand     cliconfig.NetworkCreateValues
+	networkcreateDescription = `create networkk for containers and pods`
+	_networkcreateCommand    = &cobra.Command{
+		Use:   "create [flags] [NETWORK]",
+		Short: "network create",
+		Long:  networkcreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			networkcreateCommand.InputArgs = args
+			networkcreateCommand.GlobalFlags = MainGlobalOpts
+			networkcreateCommand.Remote = remoteclient
+			return networkcreateCmd(&networkcreateCommand)
+		},
+		Example: `podman create podman1`,
```

Comment:
I think there's `network` missing -> `podman network create network-1`?

Reply:
I think we should also add the path to the config file here to make it more obvious what's happening underneath.